cmake/modules/platform/os/zos.cmake

```diff
@@ -52,11 +52,6 @@ install(TARGETS omr_ascii omr_ebcdic
 )
 
 macro(omr_os_global_setup)
```

Comment:
Please fix the copyright header in this file.

Reply:
Ping?  Should be an easy update and we can get this PR off the books.

Reply:
sorry. got buried in my inbox